packages/views/signoff.py

```diff
@@ -53,6 +55,36 @@ def signoff_package(request, name, repo, arch, revoke=False):
             return render(request, '403.html', status=403)
         signoff, created = Signoff.objects.get_or_create_from_package(
                 package, request.user)
+        signoffs = Signoff.objects.for_package(package).filter(
+                revoked__isnull=True)
+
+        signed_off_by = []
+        for s in signoffs:
+            signed_off_by.append(User.objects.get(id=s.user_id))
+
+        if  signoffs.count() >= spec.required:
+            packager = package.packager
+            if not packager:
+                toemail = [package.packager_str.split('<')[1].replace('>','')]
```

Comment:
This type of splitting is rather brittle, let's just not send out a notification if no matching ArchWeb user exists.

Reply:
Agreed, let's drop this. Maybe it should go to a generic mail address, we can add a TODO for it for now.